buku

```diff
@@ -73,9 +73,12 @@ PROMPTMSG = 'buku (? for help): '  # Prompt message string
 
 strip_delim = lambda s, delim=DELIM, sub=' ': str(s).replace(delim, sub)
 taglist = lambda ss: sorted(s.lower() for s in set(ss) if s)
-taglist_str = lambda s: delim_wrap(DELIM.join(taglist(s.split(DELIM))))
 like_escape = lambda s, c='`': s.replace(c, c+c).replace('_', c+'_').replace('%', c+'%')
 
+def taglist_str(tag_str, convert=None):
+    tags = taglist(tag_str.split(DELIM))
+    return delim_wrap(DELIM.join(tags if not convert else taglist(convert(tags))))
```

Comment:
Optionally applying a conversion to the processed tags before converting them back to a sting